<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1550</width>
    <height>957</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Canvas Eagles - Aircraft Maneuver Log</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0" rowspan="3">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0,0" columnstretch="0,1">
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QGraphicsView" name="alt_control">
        <property name="minimumSize">
         <size>
          <width>192</width>
          <height>0</height>
         </size>
        </property>
        <property name="renderHints">
         <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0" colspan="3">
         <widget class="QComboBox" name="firing_arc_selection"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="rotate_gun_left">
          <property name="text">
           <string>Rotate Left</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="gun_pos_spin">
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="prefix">
           <string>Gun Pos </string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="rotate_gun_right">
          <property name="text">
           <string>Rotate Right</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" rowspan="2" colspan="3">
         <widget class="QGraphicsView" name="firing_arc_control">
          <property name="renderHints">
           <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="log_movement_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Lock Movement</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTabWidget" name="crew_tab">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="PlanePartDamageTracker" name="wing_grp" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="PlanePartDamageTracker" name="engine_grp" native="true"/>
      </item>
      <item row="2" column="0">
       <widget class="PlanePartDamageTracker" name="fuselage_grp" native="true"/>
      </item>
      <item row="2" column="1">
       <widget class="PlanePartDamageTracker" name="tail_grp" native="true"/>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Log</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>7</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>7</number>
       </property>
       <item>
        <widget class="QLineEdit" name="date">
         <property name="placeholderText">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="turn_log">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="next_turn_btn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>End Turn</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1550</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="game_menu">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionCreate_plane"/>
    <addaction name="actionEdit_plane"/>
    <addaction name="actionLoad_Planes"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="select_plane_menu">
    <property name="title">
     <string>Select a Plane</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSet_Plane_Auto_Load_Location"/>
   </widget>
   <widget class="QMenu" name="menuGame_Effects">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Game Effects</string>
    </property>
    <addaction name="actionDamage_Chit_Effects"/>
   </widget>
   <addaction name="game_menu"/>
   <addaction name="select_plane_menu"/>
   <addaction name="menuGame_Effects"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionLoad_Planes">
   <property name="text">
    <string>Load a Plane</string>
   </property>
  </action>
  <action name="actionCreate_plane">
   <property name="text">
    <string>Create a new plane</string>
   </property>
  </action>
  <action name="actionEdit_plane">
   <property name="text">
    <string>Edit an existing plane</string>
   </property>
  </action>
  <action name="actionDogfight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dogfight</string>
   </property>
  </action>
  <action name="actionBombing_Run">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bombing Run</string>
   </property>
  </action>
  <action name="actionScouting_Mission">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scouting Mission</string>
   </property>
  </action>
  <action name="actionSet_Plane_Auto_Load_Location">
   <property name="text">
    <string>Set Auto-Load Location</string>
   </property>
  </action>
  <action name="actionSet_up_Crew">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Set-up Crew</string>
   </property>
  </action>
  <action name="actionRudder_Jam">
   <property name="text">
    <string>Rudder Jam</string>
   </property>
  </action>
  <action name="actionDamage_Chit_Effects">
   <property name="text">
    <string>Damage Chit Effects</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlanePartDamageTracker</class>
   <extends>QWidget</extends>
   <header>turn_logger/PlanePartDamageTracker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
